backends/dpdk/constants.h

```diff
@@ -28,7 +28,8 @@ const unsigned dpdk_default_table_size = 65536;
 // Maximum number of configurable timeout values
 const unsigned dpdk_learner_max_configurable_timeout_values = 8;
 const unsigned default_learner_table_size = 0x10000;
-const unsigned default_learner_table_timeout = 120;
+// default timeout values for learner table to support common protocol states
+const unsigned default_learner_table_timeout[] = {10, 30, 60, 120, 300, 43200, 120, 120};
```

Comment:
Can you add an enum/ macro for both dpdk_learner_max_configurable_timeout_values initializer and default_learner_table_timeout array size? This way, both will be in sync, manual mistakes shall be avoided (array access going out of bounds when developer updates only the max value).

Reply:
Used the max configurable value as array size